ykit.js

```diff
@@ -1,13 +1,11 @@
 module.exports = {
-    plugins: ['antd'],
-    config: {
-        exports: [
-            ['./src/index.js']            
-        ],
-        modifyWebpackConfig: function(baseConfig) {
-            baseConfig.context = require('path').resolve(__dirname);
-            baseConfig.output.prd.filename = '[name][ext]'
-            return baseConfig;
-        }
-    }
+  plugins: ["antd"],
+  config: {
+    exports: [["./src/index.js"]],
+    modifyWebpackConfig: function (baseConfig) {
```

Comment:
I don't think this is needed anymore (no webpack)

Reply:
Ah yeah, will remove